react-native-scripts/package.json

```diff
@@ -33,8 +33,9 @@
     "match-require": "^2.0.0",
     "minimist": "^1.2.0",
     "path-exists": "^3.0.0",
+    "progress": "^1.1.8",
```

Comment:
so apparently the inquirer package is also irrevocably broken on git bash, so i'm just going to step away slowly